docs/welcome/js/welcome.js

```diff
@@ -1,19 +1,22 @@
 (function() {
-  var init, setupShepherd;
 
-  init = function() {
-    return setupShepherd();
-  };
+  function init() {
+    const shepherd = setupShepherd();
 
-  setupShepherd = function() {
-    var shepherd;
-    shepherd = new Shepherd.Tour({
+    setTimeout(() => {
```

Comment:
why?

Reply:
This was to initiate a slight delay before starting the tour -- so the first entrance animation would be a bit more noticeable.